ecosystem/typescript/sdk/src/bcs/helper.ts

```diff
@@ -1,14 +1,27 @@
 // Copyright © Aptos Foundation
 // SPDX-License-Identifier: Apache-2.0
 
+import { TypeTag, TypeTagVector } from "../aptos_types";
 import { Deserializer } from "./deserializer";
 import { Serializer } from "./serializer";
 import { AnyNumber, Bytes, Seq, Uint16, Uint32, Uint8 } from "./types";
+import { serializeVector as smartSerializeVectorWithDepth } from "../transaction_builder/builder_utils";
```

Comment:
lets import `serializeVector` as is, and call the helper function `serializeVectorWithDepth` that just calls `serializeVector`

Reply:
Ok can do, but there's still a namespace collision bc there's a `serializeVector` function in the `helper.ts` file. I'll just import with `serializeVector as serializeVectorHelper`, lmk if you want it to be named something else